edgepi-leds.html

```diff
@@ -1,167 +1,188 @@
 <script type="text/javascript">
-    RED.nodes.registerType('led',{
-        category: 'EdgePi',
-        color: '#f391aa',
-        defaults: {
-            name: { value:"" },
-            transport: { value: "Local" },
-            config: { value: "Editor" },
-            method: { value: "toggleLed"},
-            ledPin: { value: "LED1" },
-            tcpAddress: { value: "" },
-            tcpPort: { value: "" }
-        },
-        inputs:1,
-        outputs:1,
-        icon: "light.svg",
-        label: function() {
-            return this.name||"led";
-        },
-        oneditprepare: function(){
-            // Query html elements
-            
-            const configTransportSelect = document.getElementById("node-input-transport");
-            const tcpField = document.querySelector(".form-row.tcp")
-            const configSelect = document.getElementById("node-input-config");
-            const methodField = document.querySelector(".form-row.method");
-            const ledPinField = document.querySelector(".form-row.ledPin");
+  RED.nodes.registerType("led", {
+    category: "EdgePi",
+    color: "#f391aa",
+    defaults: {
+      name: { value: "" },
+      transport: { value: "Local" },
+      ledState: { value: true },
+      ledPin: { value: 1 },
+      tcpAddress: { value: "" },
+      tcpPort: { value: "" },
+    },
+    inputs: 1,
+    outputs: 1,
+    icon: "light.svg",
+    label: function () {
+      return this.name || "led";
+    },
+    oneditprepare: function () {
+      const configTransportSelect = document.getElementById(
+        "node-input-transport"
+      );
+      const tcpField = document.querySelector(".form-row.tcp");
+      const ledState = document.querySelector(".form-row.ledState");
+      const ledPinField = document.querySelector(".form-row.ledPin");
+      const ledOn = document.getElementById("node-input-led-on");
+      const ledOff = document.getElementById("node-input-led-off");
```

Comment:
I am a bit confused. we have both ledState and ledOn and ledOff fields ? what is ledState for?

Reply:
based on what I read:
The ledOn and leadOff here are just the elements, and their value is `checked`. If the value of ledOn is `checked`, the ledState is `true`, and vice versa.

Reply:
Actually, the line const ledState = document.querySelector(".form-row.ledState"); is unnecessary. Removed